readthedocs/doc_builder/environments.py

```diff
@@ -782,8 +782,8 @@ def __init__(self, *args, **kwargs):
         if self.project.has_feature(Feature.USE_TESTING_BUILD_IMAGE):
             self.container_image = 'readthedocs/build:testing'
         # the image set by user or,
-        if self.config and self.config.build.image:
-            self.container_image = self.config.build.image
+        if self.config and self.config.docker_image:
```

Comment:
I assume this is backwards compatible with config changes, but just want to make sure.

Reply:
Yes. The `docker_image` property was added to the `ConfigBase` class where the other two inherit from: https://github.com/readthedocs/readthedocs.org/blob/d8651c133497ed921e13470f80e242ce099b84a4/readthedocs/config/config.py#L279-L283